public/app/plugins/datasource/influxdb/components/ConfigEditor.tsx

```diff
@@ -258,6 +276,7 @@ export class ConfigEditor extends PureComponent<Props, State> {
           <div className="gf-form-inline">
             <div className="gf-form">
               <Select
+                aria-label="Query language"
```

Comment:
Maybe we could use `aria-labelledby` here to the `Query Language` heading

Reply:
`aria-labelledby` does not seem to work for me with our `<Select/>` 😿

Reply:
:( Maybe something to add in a future PR